src/pages/costModels/costModel/addSourceWizard.tsx

```diff
@@ -49,7 +50,8 @@ const sourceTypeMap = {
 };
 
 class AddSourceWizardBase extends React.Component<Props, AddSourcesStepState> {
-  public state = { checked: {} };
+  private checkChecked = (a, b) => Object.keys(a).every(key => b[key] && a[key].selected === b[key].selected);
+  public state = { checked: {}, initialChecked: {} };
```

Comment:
Please keep the default `state` variables at the top of the component.

Typically, you'll find the life cycle methods appear next. The constructor (if any), followed by the `componentDidMount` and `componentDidUpdate` methods. For some reason, the `render` method is always last.

Reply:
Done